src/main/CommandHandler.cpp

```diff
@@ -568,22 +568,28 @@ CommandHandler::tx(std::string const& params, std::string& retStr)
         {
             // add it to our current set
             // and make sure it is valid
-            TransactionQueue::AddResult status =
-                mApp.getHerder().recvTransaction(transaction);
+            int64_t feeBumpMinFee;
+            auto status =
+                mApp.getHerder().recvTransaction(transaction, feeBumpMinFee);
+            output << "{"
+                   << "\"status\": "
+                   << "\"" << TX_STATUS_STRING[static_cast<int>(status)]
+                   << "\"";
 
-            if (status == TransactionQueue::AddResult::ADD_STATUS_PENDING)
+            switch (status)
+            {
+            case TransactionQueue::AddStatus::ADD_STATUS_PENDING:
             {
                 StellarMessage msg;
                 msg.type(TRANSACTION);
                 msg.transaction() = envelope;
                 mApp.getOverlayManager().broadcastMessage(msg);
             }
-
-            output << "{"
-                   << "\"status\": "
-                   << "\"" << TX_STATUS_STRING[static_cast<int>(status)]
-                   << "\"";
-            if (status == TransactionQueue::AddResult::ADD_STATUS_ERROR)
+            break;
+            case TransactionQueue::AddStatus::ADD_STATUS_BAD_REPLACE_BY_FEE:
+                output << " , \"min_fee\": " << feeBumpMinFee;
```

Comment:
Do we need to use a string here? Otherwise, will this work correctly with languages that don't support `int64` natively?

Reply:
+1: this is unsafe